### PR TITLE
Stats: Update dashboard widget aesthetics

### PR DESCRIPTION
**Pull request by jsnmoon**
## Proposed changes:

Before|After
-|-
image|image

- Updates the look and feel of the Jetpack Stats widget on the wp-admin dashboard.
- Removes the link for configuring Jetpack Stats.
- This design is not final, but I recommend shipping this as it's an improvement over the existing widget.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

Based heavily off of p1HpG7-l7H-p2.

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* Apply these changes to your site with the Jetpack plugin.
* Navigate to `/wp-admin/` and ensure that the Jetpack Stats widget renders with the new aesthetics.

### CODE REVIEW

**Comment by jsnmoon** [comment]
cc @jeffgolenski: I know we're still iterating on the widget design, but I wanted to push out this iteration since our existing widget looks somewhat dated.

**Review by grzegorz-cp — DISMISSED** [review]
Looks good 👍

**Review by kangzj — DISMISSED** [review]
It works as described 👍

File: projects/plugins/jetpack/_inc/client/dashboard-widget/style.scss [hunk 1]
```diff
@@ -1,3 +1,15 @@
+@mixin widget-card-background-container() {
+	background: #F9F9F6;
```
Comment (jeherve) [inline projects/plugins/jetpack/_inc/client/dashboard-widget/style.scss]: Maybe we can use variables in here, like `var( --jp-white-off )` in this case?

**Review by jeherve — DISMISSED** [review]
This is nice, it simplifies things quite a bit! 🌟 

I have a small note below, but I don't think this should be a blocker.

**Comment by jeffgolenski** [comment]
You're awesome @jsnmoon - definitely a great v.1. Could we just turn those text links under Top Posts to black + underline? Then I think we're good to go.

File: projects/plugins/jetpack/_inc/client/dashboard-widget/style.scss (hunk 1 above)
Comment (jsnmoon) [reply projects/plugins/jetpack/_inc/client/dashboard-widget/style.scss]: Done! I slightly refactored the `base-styles` package to bring in those CSS variables without extraneous styling.

**Comment by jsnmoon** [comment]
Updated link colors to black and removed the underline unless hovered/focused 🙂

**Review by jeherve — DISMISSED** [review]
This looks good to me! 🚢

**Review by jeherve — APPROVED** [review]

**Pull request merged**